frontend/src/app/modules/result-selection/components/result-selection-job-group/result-selection-job-group.component.ts

```diff
@@ -96,19 +49,7 @@ export class ResultSelectionJobGroupComponent implements OnInit {
     }
   }
 
-  private setFilteredJobGroups(tag: string): void{
-    let filteredJobGroups = [];
-    if(this.jobGroups){
-      this.jobGroups.forEach(element => {
-        if(element.tags.indexOf(tag) > -1){
-          filteredJobGroups.push(element);
-        }
-      });
-      this.filteredJobGroups = filteredJobGroups;
-    }
-  }
-
-  private sortJobGroupsByName(jobGroups: SelectableApplication[]): void{
+  updateJobGroupsAndTags(jobGroups: SelectableApplication[]): void{
     if(jobGroups!=null && jobGroups.length>0){
       this.isEmpty=false;
       jobGroups.sort((a, b) => {
```

Comment:
```suggestion
jobGroups.sort((a, b) => {
  return a.name.localeCompare(b.name);
});
```
This can also be done with the localCompare method.